fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Utils.java

```diff
@@ -579,4 +589,42 @@ public static boolean canEliminateNull(Set<ColumnRefOperator> nullOutputColumnOp
         }
         return false;
     }
+
+    public static Stream<Integer> getIntStream(RoaringBitmap bitmap) {
```

Comment:
get a better name? keyStream?

Reply:
bitmap can be considers as Set<unsigned Integer>, so getIntStream() return Stream<Integer>